osu.Game/Overlays/BeatmapSet/Buttons/PreviewButton.cs

```diff
@@ -74,6 +77,20 @@ private void load(OsuColour colours, OverlayColourProvider colourProvider)
             background.Colour = colourProvider.Background6;
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            Playing.BindDisabledChanged(_ => updateEnabledState());
+            Enabled.BindValueChanged(_ => updateEnabledState());
```

Comment:
so given that `Enabled` sets `playing.Disabled`, are these both going to fire? this is still way too complicated for how it should be. it shouldn't be this hard to disable a button. i have no clue why that boolean value has to be duplicated in 2 places and moved all over.

Reply:
I've just realized `PreviewButton` being a clickable container and exposing an `Enabled` bindable is just useless, so I removed the inheritance and it seems to have improved readability pretty well.